tests/java/org/pantsbuild/tools/junit/impl/SpecParserTest.java

```diff
@@ -50,11 +51,12 @@ public void testEmptySpecsThrows() {
     assertThat(spec.getMethods(), contains(DUMMY_METHOD_NAME));
   }
 
-  @Test public void testMethodDupsClass() throws Exception {
+  @Test public void testMethodDupsClass() {
     String specString = DUMMY_CLASS_NAME + "#" + DUMMY_METHOD_NAME;
-    SpecParser parser = new SpecParser(ImmutableList.of(DUMMY_CLASS_NAME, specString));
```

Comment:
Why did this order need to change?

Reply:
The test wasn't working before and adding the fail() made the test fail.  The order needed to be changed to get the test to pass.